pkgs/by-name/we/webcomix/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "webcomix";
+  version = "3.11.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "J-CPelletier";
+    repo = "webcomix";
+    rev = version;
+    hash = "sha256-Y16+/9TnECMkppgI/BeAbTLWt0M4V/xn1+hM4ILp/+g=";
+  };
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail "poetry>=1.2.0" poetry-core \
+      --replace-fail "poetry.masonry.api" "poetry.core.masonry.api" \
+      --replace-fail 'pytest-rerunfailures = "^11.1.2"' 'pytest-rerunfailures = "14.0"'
```

Comment:
```suggestion
      --replace-fail "poetry.masonry.api" "poetry.core.masonry.api"
```

Reply:
I am not sure about accepting this one, it wont build without these lines.